lib/parse.js

```diff
@@ -25,12 +28,15 @@ function float(string) {
   Remove line breaks, double spaces, zero-width space, asterisk, and trim the provided string
 */
 function string(string) {
+  if (string) {
```

Comment:
JS being JS, the string `""` is actually falsy, which is why the test failed...

```suggestion
  if (string === undefined) {
```

Reply:
That check is for a string not for undefined. @lazd could reverse the logic I guess and look for undefined instead?

Reply:
oh, nevermind! that was code I added and meant to delete

Reply:
Yeah in this case, if it's an empty string or not truthy, we want to return `null` so it's discarded. This isn't for parsing numbers, so `'0'` won't be an issue (yes, `'0' == false`)